core/ecschema-locaters/src/SchemaXml.ts

```diff
@@ -13,7 +13,7 @@ import { Schema } from "@itwin/ecschema-metadata";
 
 /**
  * Utility class to assist in creating serialized EC Schemas on the file system.
- * @beta
+ * @public @preview
```

Comment:
this seems unused other than in our tests. the Xml Locaters do not use it?

Reply:
The seem to use the FileLocater for finding schemas per filename. The SchemaXml namespace seem to be only used for writing. I'd suggest to make it internal.

Reply:
I made the namespace internal